docs/Get started/FAQ.md

```diff
@@ -575,11 +575,21 @@ Fleet is tested with Redis 5.0.14 and 6.2.7. Any version Redis after version 5 w
 Most likely, yes! While we'd definitely recommend keeping Fleet up to date in order to take advantage of new features and bug patches, most legacy versions should work with Redis 6. Just keep in mind that we likely haven't tested your particular combination so that you may run into some unforeseen hiccups.
 
 ## What happened to the "Schedule" page? 
-Scheduled queries are not gone! Instead the concept of a scheduled query has been merged with a saved query. 4.35 implemented an automatic migration which transitions any pre-existing scheduled query and pack into the new merged query concept.
+Scheduled queries are not gone! Instead the concept of a scheduled query has been merged with a saved query. After 4.35, there will only be one page of queries and a query can be scheduled (via familiar attributes such as "interval," "platform") or it can simply be saved to be run ad-hoc. Queries can now belong to teams or they can be a global query which every team inherits. This greatly simplifies the mental model of the product and enables us to build [exciting features](https://github.com/fleetdm/fleet/issues/7766) on top of the new unified query concept. 
```

Comment:
Would consider removing "one page of..." on the off-chance anyone might think it refers to pagination (as in limiting the # of queries). Also suggest a minor change to wording to avoid switching from singular to plural.
```suggestion
Scheduled queries are not gone! Instead, the concept of a scheduled query has been merged with a saved query. After 4.35, scheduling now happens on the queries page: a query can be scheduled (via familiar attributes such as "interval," "platform") or it can simply be saved to be run ad-hoc. A query can now belong to a team, or it can be a global query which every team inherits. This greatly simplifies the mental model of the product and enables us to build [exciting features](https://github.com/fleetdm/fleet/issues/7766) on top of the new unified query concept. 
```